cob_twist_controller/include/cob_twist_controller/interface_types/interface_type_base.h

```diff
@@ -0,0 +1,60 @@
+/*!
+ *****************************************************************
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2015 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_twist_controller
+ *
+ * \author
+ *   Author: Christoph Mark, email: christoph.mark@ipa.fraunhofer.de / christoph.mark@gmail.com
+ *
+ * \date Date of creation: June, 2015
+ *
+ * \brief
+ *   This header contains the interface description of all available
+ *   interface types (position/velocity).
+ *
+ ****************************************************************/
+#ifndef COB_CONTROL_COB_TWIST_CONTROLLER_INCLUDE_INTERFACE_TYPES_INTERFACE_TYPE_BASE_H_
+#define COB_CONTROL_COB_TWIST_CONTROLLER_INCLUDE_INTERFACE_TYPES_INTERFACE_TYPE_BASE_H_
+
+#include "ros/ros.h"
+#include "cob_twist_controller/cob_twist_controller_data_types.h"
+#include <std_msgs/Float64MultiArray.h>
```

Comment:
as these headers are now included here, we could remove them from the cob_twist_controller.h, right?